test/unit/application_validator_test.rb

```diff
@@ -3,83 +3,143 @@
 
 require "test_helper"
 require "rails_test_helper"
-
 require "packwerk/application_validator"
 
-# make sure PrivateThing.constantize succeeds to pass the privacy validity check
-require "fixtures/skeleton/components/timeline/app/models/private_thing.rb"
-
 module Packwerk
   class ApplicationValidatorTest < Minitest::Test
+    include FixtureHelper
+
```

Comment:
8949c4452d08b4e874a81ba48f969b5ac52805b0 updates existing tests to use the fixture helper.  3583992ad0a31d10f88fb9b99d150e4ff3c46ba9 then adds new tests to ApplicationValidator.  Consider looking at each commit separately to follow the changes more easily.